server_test.go

```diff
@@ -216,10 +222,14 @@ func TestServerStopFlag(t *testing.T) {
 func TestServerNewMessage(t *testing.T) {
 	t.Run("pushes new message into server.messages, returns this message", func(t *testing.T) {
```

Comment:
As we changed flow, we need to update this description too ("creates new message, returns this message")

Reply:
I just removed this.